File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeFilterPostcodeSector.scala

package org.openapitools.client.model


case class ResponseTimeFilterPostcodeSector (
    _code: String,
    _properties: ResponseTimeFilterPostcodeSectorProperties
)
object ResponseTimeFilterPostcodeSector {
    def toStringBody(var_code: Object, var_properties: Object) =
        s"""
        | {
        | "code":$var_code,"properties":$var_properties
        | }
        """.stripMargin
}

File: kotlin-spring/src/main/kotlin/org/openapitools/model/ResponseTimeFilterPostcodeSector.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.ResponseTimeFilterPostcodeSectorProperties
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size
import javax.validation.Valid

/**
 * 
 * @param code 
 * @param properties 
 */
data class ResponseTimeFilterPostcodeSector(

    @field:JsonProperty("code", required = true) val code: kotlin.String,

    @field:Valid
    @field:JsonProperty("properties", required = true) val properties: ResponseTimeFilterPostcodeSectorProperties
) {

}

